jobs/base_jobs.py

```diff
@@ -62,21 +62,35 @@
 from jobs import job_options
 import python_utils
 
-import apache_beam as beam
-from apache_beam import runners
 
-
-class JobMetaClass(type):
-    """Meta class for all of Oppia's Apache Beam jobs.
+class JobMetaclass(type):
+    """Metaclass for all of Oppia's Apache Beam jobs.
```

Comment:
Can you add instructions on usage? (similar to the docstring for swap() in test_utils.py)

Reply:
Looks good, thanks!